client/client_test.go

```diff
@@ -85,11 +86,11 @@ gnfBQoS89m3aO035henSfRFKVJkHaEoasj8hB3pwl9FGZUJp1c2JxiKzONqZhyGa
 }
 
 func TestNew(t *testing.T) {
-	mantaURL := "https://us-east.manta.joyent.com"
+	mantaURL := "https://us-central.manta.mnx.io"
 	tsgEnv := "http://tsg.test.org"
-	jpcTritonURL := "https://us-east-1.api.joyent.com"
+	jpcTritonURL := "https://us-central-1.api.mnx.io"
 	spcTritonURL := "https://us-east-1.api.samsungcloud.io"
```

Comment:
Remove all SPC references entirely.

This entire section should be reworked to not use any pre-defined URLs. It should *only* work with user supplied run-time defined account information.

Reply:
These are unit tests AFAICT. Would you like run-time defined account information for these?